images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -291,6 +291,8 @@ $markdown += New-MDHeader "Xcode" -Level 3
 $markdown += Build-XcodeTable $xcodeInfo | New-MDTable
 $markdown += New-MDNewLine
 
+$markdown += Build-XcodeCommandLineToolsSection

```

Comment:
I suggest to put this information at the begin of documentation rather than Xcode section since CLT is not related to Xcode.
![Screen Shot 2020-12-07 at 8 27 13 AM](https://user-images.githubusercontent.com/16715858/101312986-2f850b00-3866-11eb-84f5-600b136566a9.png)

Reply:
Let's put it to the Xcode section but with title "Command Line Tools: <version>"

Reply:
Sounds good. Let's put it to Xcode section before Xcode table